static/js/unread.js

```diff
@@ -574,6 +584,11 @@ export function num_unread_for_stream(stream_id) {
     return unread_topic_counter.get_stream_count(stream_id);
 }
 
+export function stream_has_any_unread_mention_messages(stream_id) {
+    const res = unread_topic_counter.get_counts();
+    return res.streams_with_mentions.includes(stream_id);
+}
```

Comment:
This seems like a bad API -- `get_counts()` is a somewhat expensive call getting global data; we definitely don't want to have a function like this that one might naturally call in a loop with that approach.

If you somehow need this API, it should just do the `const message_id of unread_mentions_counter` part of get_counts. But I suspect that in some places you're using it, the actually correct thing to do would be access the `streams_with_mentions` data in a `counts` object that you already got from `unread_topic_counter.get_counts`.

Reply:
Updated the PR, used `streams_with_mentions ` data from `counts`  object directly to get the answer for `stream_has_any_unread_mention_messages `

Thanks!